src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable no-console */
 import './_dashboard.scss';
-
-import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
-import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
-
+import { Card } from '@patternfly/react-core';
```

Comment:
almost kinda, we gotta pull each one from the dist... so card would be `import { Card } from '@patternfly/react-core/dist/js/components/Card/Card';`

Reply:
ah! ok tried again